stats.cpp

```diff
@@ -1,5 +1,55 @@
 #include "stats.h"
+#include<cmath>
+using namespace std;
 
-Stats Statistics::ComputeStatistics(const std::vector<___>& ) {
+Statistics::Stats Statistics::ComputeStatistics(const std::vector<float>& a) {
     //Implement statistics here
+     Statistics::Stats s;
+    if(a.size()==0)
```

Comment:
Make the naming self-explanatory. Variable names like a and s are not self-explanatory.